int32 data # 42